dragonphy/fpga_models/analog_slice.py

```diff
@@ -180,16 +219,23 @@ def __init__(self, filename=None, **system_values):
                 std=m.noise_rms,
                 clk=m.clk,
                 rst=m.rst,
+                ce=m.write_output,
                 lfsr_init=m.noise_seed
             )
             sample_value = m.bind_name('sample_value', sample_value_pre + sample_noise)
         else:
             sample_value = sample_value_pre
 
+        # there is a special case in which the output should not be updated:
```

Comment:
as mentioned before, there is a special case in which the ADC output is not updated and holds the same value from the last clk_adc cycle.  based on looking at logic analyzer waveforms, this really does happen from time to time